Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -5138,6 +5156,33 @@ describe(
           }
         );
       });
+
+      it("renders tileset with multiple contents", function () {
+        return Cesium3DTilesTester.loadTileset(scene, multipleContentsUrl).then(
+          function (tileset) {
+            var statistics = tileset._statistics;
+            expect(statistics.visited).toEqual(1);
+            expect(statistics.numberOfCommands).toEqual(2);
+          }
+        );
+      });
+
+      it("renders implicit tileset with multiple contents", function () {
+        var oldMaximumRequestsPerServer =
+          RequestScheduler.maximumRequestsPerServer;
+        RequestScheduler.maximumRequestsPerServer = 10;
```

Comment:
Can this be removed now after fixing the bug in https://github.com/CesiumGS/cesium/pull/9407? Same below.

Reply:
Yes! I removed it here and in Implicit3DContentSpec. Everywhere else the lines were intentional to simulate the request queue filling up